tutorials/qdrant_langchain_instrumentation_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,6089 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            vector_column_name="centered_text_vector",
</pre><p>probably makes most sense to not keep the <code>centered</code> here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>